server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionTesting.java

```diff
@@ -243,23 +247,25 @@ private ConnectionConfig reqToConnectionConfig(TestConnectionReq req) {
 
     private void testInitScript(ConnectionExtensionPoint extensionPoint,
             String schema, ConnectionConfig config) throws SQLException {
-        if (StringUtils.isEmpty(config.getSessionInitScript())) {
-            return;
-        }
         String jdbcUrl =
                 extensionPoint.generateJdbcUrl(getJdbcUrlProperties(config, schema));
 
         Properties properties = getTestConnectionProperties(config);
         properties.setProperty("socketTimeout", ConnectTypeUtil.REACHABLE_TIMEOUT_MILLIS + "");
         properties.setProperty("connectTimeout", ConnectTypeUtil.REACHABLE_TIMEOUT_MILLIS + "");
 
-        ConnectionInitializer initializer = new SessionCreatedInitializer(config, false);
+        List<ConnectionInitializer> initializers = new ArrayList<>();
+        initializers.addAll(
```

Comment:
we may add BackupInstanceInitializer first and add DataSourceInitScriptInitializer second

Reply:
Sure~